priv/stdlib/List.aes

```diff
@@ -29,9 +29,11 @@ namespace List =
     []   => abort("drop_last_unsafe: list empty")
 
 
-  function contains(e : 'a, l : list('a)) = switch(l)
```

Comment:
Please add versions of `insert_by` and `sort` (and possibly others) that instead of taking a comparison lambda, rely on the constraints

Reply:
Remember that you actually can reuse the existing definitions by just applying them with a lambda